core/src/main/java/org/geysermc/geyser/network/UpstreamPacketHandler.java

```diff
@@ -280,9 +269,33 @@ public boolean handle(ResourcePackChunkRequestPacket packet) {
             e.printStackTrace();
         }
 
+        // Check if it is the last chunk and send next pack in queue when available.
+        if (remainingSize <= ResourcePack.CHUNK_SIZE && !packsToSent.isEmpty()) {
+            sendPackDataInfo(packsToSent.pop());
```

Comment:
Is this intentional? Should this not be at the end of the method?

Reply:
Or yep, thanks, shouldn't create prs at 4 am